src/backend/distributed/utils/node_metadata.c

```diff
@@ -205,6 +205,10 @@ master_disable_node(PG_FUNCTION_ARGS)
 	CheckCitusVersion(ERROR);
 
 	workerNode = FindWorkerNode(nodeName, nodePort);
+	if (workerNode == NULL)
```

Comment:
No worries, I'm pretty sure it worked back when you wrote it, I've since messed with this function a little pretty sure I introduced this in a recent PR.